x-pack/test/alerting_api_integration/spaces_only/tests/alerting/get_execution_log.ts

```diff
@@ -108,7 +108,7 @@ export default function createGetExecutionLogTests({ getService }: FtrProviderCo
       const { body: createdRule } = await supertest
         .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule`)
         .set('kbn-xsrf', 'foo')
-        .send(getTestRuleData({ schedule: { interval: '15s' } }))
+        .send(getTestRuleData({ enabled: false, schedule: { interval: '15s' } }))
```

Comment:
This is testing that the API returns an empty response when there is no event log data for a rule. Previously there was a race condition where it would sometimes get event log data so creating this rule as disabled should prevent any executions from happening.